client/examples/workflow-webapp/src/main.ts

```diff
@@ -33,17 +33,19 @@ const filePath = urlParameters.path;
 
 // In the Eclipse Integration, port is dynamic, as multiple editors
 // and/or Eclipse Servers may be running in parallel (e.g. 1/Eclipse IDE)
-const port = parseInt(urlParameters.port);
+const port = parseInt(urlParameters.port, 10);
 const id = "workflow";
 const name = "Workflow Diagram";
 const websocket = new WebSocket(`ws://localhost:${port}/${id}`);
 
-const container = createContainer();
+const clientId = urlParameters.client ? urlParameters.client : ApplicationIdProvider.get();
+console.log("ApplicationId", ApplicationIdProvider.get());
```

Comment:
Why do we log the `ApplicationId` here?

Reply:
My mistake, forgot to remove it, had it in for testing.